cowprotocol/cowswap
# docs: add documentation for UTM module

# Summary

Documents the UTM module (just because is the simplest!)

# To Test
- Verify the README --> https://github.com/cowprotocol/cowswap/blob/docs/utm-module/src/modules/utm/utm-module.md
- Check the changes in the overall README --> now it has a link to the UTM docs, and some structure to add the module links as we document it https://github.com/cowprotocol/cowswap/blob/docs/utm-module/docs/architecture-overview.md
- Give feedback if you feel this is a good way to document what a module does. This didn't need much documentation, so I was just explained it don't depend on other modules, and what's the facade. Other modules might incorporate some additional things (like screenshots, sequence diagrams, etc). I think we should be reasonable and not add for the sake of adding, just to make it clear for someone to understand

![image](https://github.com/cowprotocol/cowswap/assets/2352112/abe9e8ef-b621-4125-bb86-99c36ebc73eb)
